### DISCUSSION TITLE
Analytics issue

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3023] [Hubs Cloud Questions] [answered]
Hello 
I have connect my hubs to google analytics , but there is no record of data . Is someone have this issue with a hub entreprise and custom client ? 
Thx

**Comment by emclaren** [comment]
Hi there, someone else reported an issue with setting up Google Analytics recently, do you think that this is similar to the issue you were experiencing: https://github.com/mozilla/hubs/discussions/3101

I believe they were able to get their analytics up-and-running in the end. Let us know!

**Comment by robinkwilson** [comment, marked as answer]
Added a **FOR NOW** solution to the github issue via the Admin Panel: https://github.com/mozilla/hubs-cloud/issues/117

Follow the issue to see the complete fix merged soon.
